G/GasModels/Compat.toml

```diff
@@ -1,18 +1,18 @@
-[0]
-JSON = "0.6-0"
-JuMP = "0.17-0.18"
-MathProgBase = "0.5.4-0"
-
 ["0-0.1"]
 julia = "0.5-0"
 
 ["0-0.3.1"]
 Compat = "0.17-2"
 
+["0-0.3.4"]
+JSON = "0.6-0"
+JuMP = "0.17-0.18"
+MathProgBase = "0.5.4-0"
```

Comment:
Is this requirement not needed anymore? Consider having `[compat]` for all deps in future releases.